### ISSUE TITLE
Add statistics about the loaded database

### ISSUE THREAD

**Issue by dkratzert** [dkratzert/StructureFinder#19]
* Space groups
* dates
* authors
* Any other ideas?

Make them clickable in order to get all e.g. P4 structures or all from author x.

**Comment by dwpaley** [comment]
Z' is a simple and useful one. Hydration/solvation status potentially useful but not so simple, although the information "should" be derivable from the moiety formula.

**Comment by dkratzert** [comment]
> Z' is a simple and useful one. Hydration/solvation status potentially useful but not so simple, although the information "should" be derivable from the moiety formula.

Good idea. I will first implement the low-hanging fruits, but Z' is one.

**Comment by dkratzert** [comment]
Other random ideas from users:

- Crystal systems; Bravais centering.
- Specific symmetry operations (screw axes; glides; diamond glides; etc)
- Chiral vs centro vs non-centro.
- Resolution?
- Bond accuracy/precision?  
- R-values?
- Unit cell volumes? Unit cell axes?
- Unit cells in non-standard settings (P2(1)/a, C2/c that should rather be I2/a, etc)
- Twinning? HKLF4 vs HKLF 5 and TWIN command other than inversion?
- Disorder? Anything that has a PART command or a negative PART command?
- Structures with verification replies?
- Restraints used (SADI, SAME, SIMU, RIGU, ISOR, FLAT, CHIV, …), constraints used (EADP, EXYZ, AFIX 66, etc)
- Wavelengths?
- Squeeze or Mask used (ABIN command)
